source/Octopus.Client/OctopusAsyncClient.cs

```diff
@@ -27,25 +26,19 @@ namespace Octopus.Client
     public class OctopusAsyncClient : IOctopusAsyncClient
     {
         private static readonly ILog Logger = LogProvider.For<OctopusAsyncClient>();
-
-        readonly OctopusServerEndpoint serverEndpoint;
-        readonly JsonSerializerSettings defaultJsonSerializerSettings = JsonSerialization.GetDefaultSerializerSettings();
+     
+        private readonly OctopusServerEndpoint serverEndpoint;
+        private readonly JsonSerializerSettings defaultJsonSerializerSettings = JsonSerialization.GetDefaultSerializerSettings();
         private readonly HttpClient client;
         private readonly CookieContainer cookieContainer = new CookieContainer();
         private readonly Uri cookieOriginUri;
         private readonly bool ignoreSslErrors = false;
-        bool ignoreSslErrorMessageLogged = false;
-        private readonly string rootDocumentUri;
-        private OctopusClientOptions clientOptions;
-        private bool signedIn = false;
-        private bool IsAuthenticated => (signedIn || !string.IsNullOrEmpty(this.serverEndpoint.ApiKey));
-
-        // Use the Create method to instantiate
-        private OctopusAsyncClient(OctopusServerEndpoint serverEndpoint, OctopusClientOptions options, bool addCertificateCallback)
+        private bool ignoreSslErrorMessageLogged = false;
+        private string antiforgeryCookieName = null;
+
+        public OctopusAsyncClient(OctopusServerEndpoint serverEndpoint, OctopusClientOptions options, bool addCertificateCallback)
```

Comment:
Given that `Create` is still required to do async work ( 😢 ) this should probably still be private...